### PR TITLE
Update RELEASE_INSTRUCTIONS.md

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: tool/RELEASE_INSTRUCTIONS.md [hunk 1]
```diff
@@ -213,6 +215,11 @@ to add DevTools release notes to Flutter website and test them in DevTools.
 [g/flutter-internal-announce](http://g/flutter-internal-announce) with a link to
 the new release notes.
 
+## Dev release into the Dart SDK master branch
+
+To publish a dev release follow just section [Update the DevTools hash in the Dart SDK](#update-the-devtools-hash-in-the-dart-sdk)
+of the full release process. The dev release can be later converted to a full release by all other steps for the selected commit.
```
Comment (kenzieschmoll) [inline tool/RELEASE_INSTRUCTIONS.md:221]: remove last sentence. There is no way to "convert" a dev release into a full release. A full release is independent of the dev release.

Comment (polina-c) [reply tool/RELEASE_INSTRUCTIONS.md:221]: I thought we can use the same commit for full release, if it is already in DEP of SDK. Am I wrong?

Comment (kenzieschmoll) [reply tool/RELEASE_INSTRUCTIONS.md:221]: This commit will not include the updated DevTools version that is part of doing a full release.

Comment (kenzieschmoll) [reply tool/RELEASE_INSTRUCTIONS.md:221]: we do not want to do a release into flutter stable or beta with a DevTools version like `2.24.0-dev.15`, so for full releases, we do a release version update (e.g. 2.24.0)

Comment (polina-c) [reply tool/RELEASE_INSTRUCTIONS.md:221]: got it! thanks

**Review by kenzieschmoll — APPROVED** [review]

**Pull request merged**